proxy/api/src/http/identity.rs

```diff
@@ -54,20 +54,23 @@ mod handler {
         ctx: context::Unsealed,
         input: super::CreateInput,
     ) -> Result<impl Reply, Rejection> {
-        if let Some(identity) = session::current(ctx.state.clone(), &ctx.store)
-            .await?
-            .identity
-        {
+        if let Some(session) = session::get_current(&ctx.store)? {
             return Err(Rejection::from(error::Error::from(
-                coco::state::Error::already_exists(identity.urn),
+                coco::state::Error::already_exists(session.identity.urn),
             )));
         }
 
         let id = identity::create(&ctx.state, &input.handle).await?;
 
-        session::set_identity(&ctx.store, id.clone())?;
+        session::initialize(&ctx.store, id.clone())?;
+        let mut cookie = ctx.auth_cookie.write().await;
+        *cookie = Some("chocolate".into());
```

Comment:
When are we going to create unique ones?

Reply:
@MeBrei is currently working on this as an immediate follow-up.